### ISSUE TITLE
Need to set top of browser scroll area to execute command card.

### ISSUE THREAD

**Issue by GrahamDumpleton** [vmware-tanzu/octant#738]
**What steps did you take and what happened:**

Go to pod details.

Scroll down to `EXECUTE COMMAND`, such that the top of the `Container` card is off the top of the browser.

![image](https://user-images.githubusercontent.com/507637/75500212-664d0b80-5a20-11ea-9c1c-8efdeb284f11.png)

Click on `EXECUTE COMMAND`.

The execute command card replaces temporarily the container card, but in doing that, it is now off screen and you need to scroll back up wards to find it and enter the command.

![image](https://user-images.githubusercontent.com/507637/75500247-811f8000-5a20-11ea-81d4-6c44ec56442b.png)

**What did you expect to happen:**

The page should reset the scroll position so the top of the execute command card is at the top of the displayed region of the page.

**Anything else you would like to add:**

Nope.

**Environment:**

- Octant version (use `octant version`): 

```
$ octant version
Version:  0.10.2
Git commit:  438ce3d52088c1e3e15aaf9817ba71e3c85255f5
Built:  2020-02-15T02:33:23Z
```

- Kubernetes version (use `kubectl version`):

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-13T18:08:14Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-11T18:07:13Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
```

- OS (macOS 10.15, Windows 10, Ubuntu 19.10 etc):

```
$ uname -a
Darwin grumpy-at-home.fritz.box 19.3.0 Darwin Kernel Version 19.3.0: Thu Jan  9 20:58:23 PST 2020; root:xnu-6153.81.5~1/RELEASE_X86_64 x86_64
```

Also affects separate web browser when accessing Octant as web service.

**Comment by wwitzel3** [comment]
This will be fixed when we do #664

**Comment by GuessWhoSamFoo** [comment]
`Execute Command` has been removed in favor of a terminal tab that creates a new terminal upon selecting. Closed via 9d5a9f92942b07fc9aec31b11f74e85469120043
